packages/flutter/lib/src/material/icons.dart

```diff
@@ -10287,7 +10286,7 @@ class Icons {
   /// <i class="material-icons-round md-36">insert_chart</i> &#x2014; material icon named "insert chart" (round).
   static const IconData insert_chart_rounded = IconData(0xf819, fontFamily: 'MaterialIcons');
 
-  /// <i class="material-icons md-36">insert_chart_outlined</i> &#x2014; material icon named "insert chart outlined".
+  /// <i class="material-icons-outlined md-36">insert_chart</i> &#x2014; material icon named "insert chart" (outlined).
```

Comment:
Ah I see, so we changed that one that I mentioned above?

Reply:
Yeah I double checked with the codepoints, it should've been the outlined variant from the start